cmd/lookout-sdk/event.go

```diff
@@ -32,9 +32,7 @@ type EventCommand struct {
 	RevFrom    string `long:"from" default:"HEAD^" description:"name of the base revision for event"`
 	RevTo      string `long:"to" default:"HEAD" description:"name of the head revision for event"`
 	ConfigJSON string `long:"config-json" description:"arbitrary JSON configuration for request to an analyzer"`
-	Args       struct {
-		Analyzer string `positional-arg-name:"analyzer" description:"gRPC URL of the analyzer to use"`
-	} `positional-args:"yes" required:"yes"`
+	Analyzer   string `long:"analyzer" default:"ipv4://localhost:9930" description:"gRPC URL of the analyzer to use"`
```

Comment:
With this change the new release will break the usage from/to
- `lookout-sdk review ipv4://localhost:10302`
- `lookout-sdk review --analyzer ipv4://localhost:10302`

right?
Is there a need for this, or could we just make the argument optional to keep compatibility?

Reply:
The reason why it's changed is in the commit message: https://github.com/src-d/lookout/pull/389/commits/3275303db234dd6e4e314c2787683a3bdec48d60

I would prefer to use `--analyze` as a recommended way to change address. But return positional argument for compatibility. WDYT?

Reply:
I see your point. Even if it makes the code ugly, I think we should prioritize compatibility and keep the argument.
And if we keep the argument having also `--analyzer` to do the same thing will lead to unnecessary confusion.

I would just go with the optional argument, checking in our code if it's missing to use the default, and adding to the argument description `  (default: ipv4://localhost:9930)`